Makefile

```diff
@@ -1,6 +1,22 @@
 artifact_name       := account-validator-api
 version             := "unversioned"
 
+##### Start of variable block for top of Makfile
+dependency_check_base_suppressions:=common_suppressions_spring_6.xml
+
+# dependency_check_suppressions_repo_branch
+# The branch of the dependency-check-suppressions repository to use
+# as the source of the suppressions file.
+# This should point to "main" branch when being used for release,
+# but can point to a different branch for experimentation/development.
+dependency_check_suppressions_repo_branch:=main
```

Comment:
Will need to point this at the feature branch, its still not merged

Reply:
+1

Reply:
True. I will change it to the ju-21-migration branch